src/classes/speciessite.cpp

```diff
@@ -278,7 +278,7 @@ Site *SpeciesSite::createFromParent() const
     if (originMassWeighted_)
     {
         double massNorm = 0.0;
-        for (auto m = 0; m < originIndices.nItems(); ++m)
+        for (auto m = 0; m < originIndices.size(); ++m)
```

Comment:
```suggestion
        for (const auto &index : originIndices)
```

Now here's a case where a standard `mean` algorithm would come in handy.